minitest/use_cases/company.py

```
@@ -0,0 +1,41 @@
+    return company
+
+def create_company(user_id, company_name, company_address, total_staff):
+    name_repo = NameRepository()
+    user = name_repo.get_user_by_uuid(user_id)
+    if user.role != 'boss':
+        raise ValidationError('사용자는 관리자여야 합니다')
+    if company_repo.find_by_user_id(user_id):
+        raise ValidationError('사용자는 이미 회사에 속해 있습니다')
+    company = Company(
+        id=uuid.uuid4(), 
+        user_id=user, 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)